libs/librepcb/libraryeditor/libraryeditor.h

```diff
@@ -126,10 +126,12 @@ class LibraryEditor final : public QMainWindow,
   // Operator Overloadings
   LibraryEditor& operator=(const LibraryEditor& rhs) = delete;
 
+public slots:
+  void closeTabIfOpen(const FilePath& fp) noexcept;
```

Comment:
This method does not need to be public, it's never called (directly) from outside this class. In addition, it does not need to be marked as a slot, since C++11 also "normal" methods can be used as a slot when using the overload of `QObject::connect()` which takes function pointers. So I would suggest to move this method below `editDeviceTriggered()`, then the order corresponds to the `connect()` calls in the constructor :)